WordPress/Classes/ViewRelated/Notifications/Views/NoteBlockTableViewCell.swift

```diff
@@ -11,7 +11,11 @@ class NoteBlockTableViewCell: WPTableViewCell {
     ///
     /// - Note: After setting this property you should explicitly call `refreshSeparators` from within `UITableView.willDisplayCell`.
     ///
-    @objc var isBadge = false
+    @objc var isBadge = false {
+        didSet {
+            separatorsView.backgroundColor = WPStyleGuide.Notifications.blockBackgroundColorForRichText(isBadge)
```

Comment:
Why do we need that?

Reply:
The default background color for the table cell is `.listForeground` which results in the gray background colors in dark mode and the badge background color is clear. So this resets all the cells to have a clear background when they're badges.

Reply:
And since the `separatorsView` is set to the backgroundView for the cell we need to change its color instead of of the cells backgroundColor